src/StatsBase.jl

```diff
@@ -188,10 +188,11 @@ export
     islinear,
     leverage,
     loglikelihood,
-    meanresponse,
     modelmatrix,
     mss,
     response,
+    responsename,
+    meanresponse,
```

Comment:
AFAICT names in this list are sorted lexicographically, so better keep the current order.